docs/resources/secmaster_indicator.md

```diff
@@ -0,0 +1,123 @@
+---
+subcategory: "SecMaster"
+---
+
+# huaweicloud_secmaster_indicator
+
+Manages a SecMaster indicator resource within HuaweiCloud.
+
+~> This resource can only be used in region **cn-east-3** for now.
+
+## Example Usage
+
+```hcl
+variable "workspace_id" {}
+
+resource "huaweicloud_secmaster_indicator" "test" {
+  workspace_id = var.workspace_id
+  name         = "test_update"

```

Comment:
use a variable or other names will be fine
```suggestion
  name         = "demo"

```

Reply:
fixed